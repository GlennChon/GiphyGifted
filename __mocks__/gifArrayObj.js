export const gifArrayObj = [
  {
    images: {
      fixed_height_downsampled: { url: "fhdUrl" },
      fixed_width_downsampled: { url: "fwdUrl" },
      fixed_height_still: { url: "fhsUrl" },
      fixed_width_still: { url: "fwsURL" }
    }
  },
  {
    images: {
      fixed_height_downsampled: { url: "fhdUrl" },
      fixed_width_downsampled: { url: "fwdUrl" },
      fixed_height_still: { url: "fhsUrl" },
      fixed_width_still: { url: "fwsURL" }
    }
  },
  {
    images: {
      fixed_height_downsampled: { url: "fhdUrl" },
      fixed_width_downsampled: { url: "fwdUrl" },
      fixed_height_still: { url: "fhsUrl" },
      fixed_width_still: { url: "fwsURL" }
    }
  }
];
