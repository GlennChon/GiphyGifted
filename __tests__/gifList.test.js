import React from "react";
import GifList from "../src/components/gifList";
import { gifArrayObj } from "../__mocks__/gifArrayObj";

describe("<GifList/>", () => {
  it("testing component snapshot", () => {
    const wrapper = shallow(<GifList gifs={gifArrayObj} />);
    expect(wrapper).toMatchSnapshot();
  });
});
