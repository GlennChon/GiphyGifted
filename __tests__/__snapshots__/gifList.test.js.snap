// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GifList/> testing component snapshot 1`] = `
<Fragment>
  <div
    key="0"
  >
    <Gif
      gif={
        Object {
          "images": Object {
            "fixed_height_downsampled": Object {
              "url": "fhdUrl",
            },
            "fixed_height_still": Object {
              "url": "fhsUrl",
            },
            "fixed_width_downsampled": Object {
              "url": "fwdUrl",
            },
            "fixed_width_still": Object {
              "url": "fwsURL",
            },
          },
        }
      }
      isAutoPlay={false}
      type="fixed-width"
    />
  </div>
  <div
    key="1"
  >
    <Gif
      gif={
        Object {
          "images": Object {
            "fixed_height_downsampled": Object {
              "url": "fhdUrl",
            },
            "fixed_height_still": Object {
              "url": "fhsUrl",
            },
            "fixed_width_downsampled": Object {
              "url": "fwdUrl",
            },
            "fixed_width_still": Object {
              "url": "fwsURL",
            },
          },
        }
      }
      isAutoPlay={false}
      type="fixed-width"
    />
  </div>
  <div
    key="2"
  >
    <Gif
      gif={
        Object {
          "images": Object {
            "fixed_height_downsampled": Object {
              "url": "fhdUrl",
            },
            "fixed_height_still": Object {
              "url": "fhsUrl",
            },
            "fixed_width_downsampled": Object {
              "url": "fwdUrl",
            },
            "fixed_width_still": Object {
              "url": "fwsURL",
            },
          },
        }
      }
      isAutoPlay={false}
      type="fixed-width"
    />
  </div>
</Fragment>
`;
